homepage/models.py

```diff
@@ -48,11 +49,39 @@ def __str__(self):
 
 
 class AppUser(models.Model):
+    # This field is the built-in django user model
     user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
 
     def __str__(self):
         return self.user.username
 
+    def toggle_user_activation(self):
+        self.user.is_active = not(self.user.is_active)
+
+    @staticmethod
+    # Creates a new appUser (without superuser permmisions) and saves it in the DB
+    def create_appUser(username, email, password):
+        appUser = AppUser()
+        user = User.objects.create_user(username, email, password)
+        appUser.user = user
+
+        appUser.save()
+        return appUser
+
+    @staticmethod
+    def get_all_appUsers():
+        return list(AppUser.objects.all())
+
+    @staticmethod
+    def find_appUser(username_ToFind):
+        allAppUsers = AppUser.get_all_appUsers()

```

Comment:
I was trying to do something like:
AppUser.objects.filter(user.username = username_ToFind)

however its not possible in this way (username is an attribute of User and not AppUser)

After some tests i found another way to do it, so it will be changed in my next commit.